airflow/.github/dependabot.yml

```diff
@@ -0,0 +1,12 @@
+# To get started with Dependabot version updates, you'll need to specify which
```

Comment:
is this required as well?
As far as I remember this dependabot thing is more annoying than useful